docs/src/main/sphinx/release/release-372.md

```diff
@@ -0,0 +1,90 @@
+# Release 372 (23 Feb 2022)
+
+## General
+
+* Remove support for reserved memory pool. Configuration property
+  `experimental.reserved-pool-disabled` can no longer be used. ({issue}`6677`)
+* Ensure memory is released completely after query completion. ({issue}` 11030`)
+
+## Base-JDBC connector library
+
+* Fix spurious query failures when metadata cache is not enabled, and data
+  access depends on the session state. ({issue}`11068`)
+
+## Accumulo connector
+
+* Fix incorrect date results on `date` column type. ({issue}`11055`)
+
+## Cassandra connector
+
+* Fix incorrect results when filtering partition keys without projections.
+  ({issue}`11001`)
+
+## ClickHouse connector
+
+* Fix spurious query failures when metadata cache is not enabled, and extra
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## Druid connector
+
+* Fix spurious query failures when metadata cache is not enabled, and extra
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## Hive connector
+
+* File status cache content is invalidated when writing through Trino.
+  ({issue}`10621`)
```

Comment:
this describes the fix itself - not the issue that was fixed. i.e. what users would observe after this change.

cc: @findinpath suggestion?

Reply:
Ah, I see. The 3rd entry here "Fix incorrect query results" is the better release note. @mosabua This one should be dropped.

Reply:
done